src/components/user/repo/account.repo.js

```diff
@@ -0,0 +1,32 @@
+const accountModel = require('../../../services/account/model');
+
+const getUserAccount = async (userId, app) => {
+	const account = await accountModel.findOne({ userId });
+	return account;
+};
+
+const deleteUserAccount = async (userId, app) => {
+	const result = await accountModel.deleteOne({ userId });
+	return result;
+};
+
+const replaceUserAccountWithTombstone = async (userId, app) => {
```

Comment:
do we need account tombstones? I think its enough to keep the account in the bin, and delete the original account to ensure it is not accidently used for some login